gcloud/bigquery/client.py

```diff
@@ -96,3 +97,20 @@ def dataset(self, name):
         :returns: a new ``Dataset`` instance
         """
         return Dataset(name, client=self)
+
+    def load_from_storage(self, name, destination, *source_uris):
```

Comment:
Would `load_table_from_storage` be better?  Or does this method (and the `copy_table` one in #1064) need to have the word `job` (maybe `async`) embedded?